lang/LangSource/PyrMathOps.cpp

```diff
@@ -227,7 +227,7 @@ int doSpecialUnaryArithMsg(VMGlobals* g, int numArgsPushed) {
             SetTrue(a);
             break;
         case opAsInteger:
-            SetTagRaw(a, tagInt);
```

Comment:
Can we remove the `SetTagRaw` function entirely? it seems like your change eliminates nearly all of them (sorry if you already did this and I missed it)

Reply:
Yes it removed it completely The approach doesn't really make sense with nan boxing on 64-bit systems. Besides, it only saved a couple of shifts and bit ops.